docs/nightly_releases.md

```diff
@@ -1,12 +1,5 @@
 # Nightly releases
 
-> [!WARNING]
-> This is still under development! See
-> https://github.com/nod-ai/shark-ai/issues/400.
->
-> These instructions will be converted into a user guide once stable packages
-> are published to PyPI: <https://github.com/nod-ai/shark-ai/issues/359>.
-
 Nightly releases are uploaded to
 https://github.com/nod-ai/shark-ai/releases/tag/dev-wheels.
```

Comment:
Note to self (_maybe_ worth a TODO in the doc?): mention the consolidated IREE ecosystem release index page when it exists: https://github.com/iree-org/iree/issues/19193